features/step_definitions/product_categories_steps.rb

```diff
@@ -1,8 +1,10 @@
 Given("the following products exist within a specific restaurant and category") do |table|
   table.hashes.each do |product|
     restaurant = Restaurant.find_by(name: product[:restaurant])
+    menu = FactoryBot.create(:menu, name: product[:menu], restaurant: restaurant)
```

Comment:
Possible refractoring for 'FactoryBot' -- it should be sufficient to just write 'create'

Reply:
Good point @lcegit Not a blocker atm though.